hotel-management/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react';
+
+import './App.css';
+
+const App = () => {
+  return (
+    <>
```

Comment:
Please recheck ur eslint actually work, if it work you should be warned by this one, redundant fragment